pkgs/by-name/libresoc/soc.nix

```diff
@@ -0,0 +1,65 @@
+{
+  python39Packages,
+  fetchFromLibresoc,
+  yosys,
+  libresoc-c4m-jtag,
+  libresoc-ieee754fpu,
+  libresoc-openpower-isa,
+  nmigen-soc,
+  power-instruction-analyzer,
+  pytest-output-to-files,
+}:
+with python39Packages;
+  buildPythonPackage rec {
+    name = "soc";
+    pname = name;
+    version = "unstable-2024-03-31";
+
+    src = fetchFromLibresoc {
+      inherit pname;
+      rev = "2a66fe18cd77dd5533c65930d1b241cf6faac455"; # HEAD @ version date
+      hash = "sha256-yJshQYf8V0CB2vPCmWLlnxXMhi/sPXiLKzOn6cqgmxw=";
+      fetchSubmodules = false;
```

Comment:
The way `fetchFromLibresoc` is implemented, it will not pass `fetchSubmodules = false;` to `fetchgit`. Please either remove this, or remove the ellipsis from the arguments of `fetchFromLibresoc` (see comment over at its definition).